src/main/java/com/azure/cosmos/examples/changefeed/SampleConfigurations.java

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.cosmos.examples.changefeed;
+
+import com.google.common.base.Strings;
```

Comment:
Can we avoid taking any other dependencies and just the ones in Java (or Cosmos SDK)? As Mo commented, people tend to copy/paste the sample code in their apps and we should try to avoid bringing new dependencies that are not required for their app to execute.